src/sentry/api/endpoints/authenticator_index.py

```diff
@@ -13,14 +14,28 @@ def get(self, request):
 
         # Currently just expose u2f challenge, not sure if it's necessary to list all
         # authenticator interfaces that are enabled
+        # this is a weird endpoint, looks like its only being used in test_user_authenticators_index.py
         try:
             interface = Authenticator.objects.get_interface(request.user, "u2f")
             if not interface.is_enrolled():
                 raise LookupError()
         except LookupError:
             return Response([])
 
-        challenge = interface.activate(request._request).challenge
+        orgs = request.user.get_orgs()
+        webauthn_ff = (
+            True
+            if any(
+                features.has("organizations:webauthn-login", org, actor=request.user)
+                for org in orgs
+            )
+            or any(
```

Comment:
you shouldn't have to loop through orgs twice, you can check both feature flags in one loop.